src/init.cpp

```diff
@@ -1827,13 +1827,13 @@ bool AppInitMain(Config &config,
         }
     }
     // Can only remove older after creating new one since there are LogPrintf in there!
-    if (RenamedLogfile != "") LogPrintf("Renamed old log file %s to %s\n",DEFAULT_DEBUGLOGFILE, RenamedLogfile);
+ /*   if (RenamedLogfile != "") LogPrintf("Renamed old log file %s to %s\n",DEFAULT_DEBUGLOGFILE, RenamedLogfile);
     logger.RemoveOlderDebugFiles();
 
 
     if (!logger.m_log_timestamps) {
         LogPrintf("Startup time: %s\n", FormatISO8601DateTime(GetTime()));
-    }
+    } */
```

Comment:
Startup time printout not part of renaming

Reply:
oops! fixed both